sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/models/InnerError.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
```

Comment:
I'm confused. I thought this whole SDK was handwritten? Is this just a copy-paste error?

Reply:
Yes , got these to generate the ErrorResponseException from generated code. I will fix it !